easybuild/easyconfigs/j/jupyter-server/jupyter-core-5.3.1_fix_jupyter_path.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/jupyter_core/paths.py b/jupyter_core/paths.py
```

Comment:
please add a patch description (and author)
you'll need to update the checksums afterwards